module/Company/src/Company/Service/Company.php

```diff
@@ -627,6 +731,29 @@ public function getAllCategoriesById($categoryID)
 
         return $package;
     }
+
+
+    /**
+     * Returns a persistent label
+     *
+     * @param mixed $labelID
+     */
+    public function getAllLabelsById($labelID)
+    {
+        if (!$this->isAllowed('edit')) {
+            $translator = $this->getTranslator();
+            throw new \User\Permissions\NotAllowedException(
+                $translator->translate('You are not allowed to edit packages')
+            );
+        }
+        if (is_null($labelID)) {
+            throw new \Exception('Invalid argument');
+        }
+        $package = $this->getCategoryMapper()->findAllCategoriesById($labelID);
+
+        return $package;
```

Comment:
You can simply return this immediately, no need to store it in a variable in between.

Reply:
Agreed